src/Arcus.WebApi.Logging/RequestTrackingMiddleware.cs

```diff
@@ -143,18 +145,31 @@ private static async Task<string> SanitizeRequestBodyAsync(Stream requestStream)
                 return "Request body could not be tracked because stream is not seekable";
             }
 
-            long originalPosition = requestStream.Position;
-            if (requestStream.Position != 0)
+            string contents = null;
+            long? originalPosition = null;
+
+            try
             {
-                requestStream.Seek(0, SeekOrigin.Begin);
-            }
+                originalPosition = requestStream.Position;
+                if (requestStream.Position != 0)
+                {
+                    requestStream.Seek(0, SeekOrigin.Begin);
+                }
 
-            var reader = new StreamReader(requestStream);
-            string contents = await reader.ReadToEndAsync();
+                var reader = new StreamReader(requestStream);
+                contents = await reader.ReadToEndAsync();
+            }
+            catch (Exception ex)
+            {
+                _logger.LogWarning(ex, "Unable to get request body for request tracking");
```

Comment:
I figured to write a warning but happy to remove if need be @fgheysels

Reply:
I don't know what I 'd do.  Log it as a wzrning, or log that in the 'requestbody' custom property ....

Reply:
(The latter one would be in line with the other loggings)

Reply:
So in that case, you would add the text field in the custom property instead then?

Reply:
Done